test/other/metadce/test_metadce_hello_O3.size

```diff
@@ -1 +1 @@
-1742
+1739
```

Comment:
Why aren't these sizes changing much? Removing the names section should make a larger difference I'd think?

Reply:
The name section was already being stripped, even with the old code.

This change just enables the the import and export section to be minified.